.github/workflows/deploy-staging.yml

```
@@ -0,0 +1,30 @@
+name: Deploy to staging
+
+on:
+  push:
+    branches:
+      - development
+
+jobs:
+  deploy-staging:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install SSH key for deploying
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.DEPLOY_SSH_PRIVATE_KEY }}
+          known_hosts: ${{ secrets.DEPLOY_KNOWN_HOSTS_STAGING }}
+          config: ${{ secrets.DEPLOY_SSH_CONFIG }}
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+      - name: Install ansible and deploy to staging
+        run: |
+          python -m pip install --user ansible-core==2.11.1
+          cd deploy
+          echo ${{ secrets.ANSIBLE_VAULT_PASSWORD }} >> ansible-vault-password.txt
```

```suggestion
          echo ${{ secrets.ANSIBLE_VAULT_PASSWORD }} > ansible-vault-password.txt
```

(This hardly matters since the file doesn't exist prior to this command; it's just that using the "append" operator insinuates that it does.)